CONTRIBUTING.md

```diff
@@ -49,6 +50,15 @@ As a contributor, the expectation is that you can use Git and GitHub comfortably
 
 3.  It is common practice to name your branch according to what it is fixing or updating. Before submitting these changes, verify they work. You can include screenshots of the before and after, if the changes are HTML/CSS based, and drop those images into your pull request for further verification. Please try to maintain the look of the site as we currently have it.
 
+#### Verifying Your Changes
+
+  Verify your changes are correct before submitting a pull request by rendering your html doc.  Because the project uses Ajax, you must have a local web server for this to work properly.  If you have OS X:
+
+    $  cd myrepo
+    $  python -m SimpleHTTPServer
+    *  Running on [http://0.0.0.0:8000](http://0.0.0.0:8000/)
+
+
 4.  Once you have submitted a pull request, we will review it. Post any changes or suggestions for review or opinion and handle accordingly. Not every pull request will be added to our database. Follow the request to keep up to date on its status.
```

Comment:
line 69: You could replace the words "..at which point.." with "then"  to cut down on words, for quicker reading.